framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3,6 +3,8 @@
 // The imports from com.sun are all @jdk.Exported and therefore somewhat safe to use.
 // Try to avoid using non-@jdk.Exported classes.
 
+import static javax.tools.Diagnostic.Kind.ERROR;
```

Comment:
Am I missing where this is used in this file?

Reply:
It's used in a `switch` statement where the import isn't needed.  Thanks for catching that.